src/options/options.js

```diff
@@ -5,24 +5,47 @@ import { error } from "../libs/logging";
 function saveOptions(e) {
 	e.preventDefault();
 	storage.sync.set({
+		proplate: {
+			enabled: document.querySelector("#proplates-enabled").checked,
+			statusVisibility: document.querySelector("#status").checked,
+			notificationVisibility: document.querySelector("#notification").checked,
+			accountVisibility: document.querySelector("#account").checked,
+			conversationVisibility: document.querySelector("#conversation").checked,
+		},
+		hoverCard: {
+			enabled: document.querySelector("#hovercards-enabled").checked,
+			stats: document.querySelector("#card-stats").checked,
+			privateNote: document.querySelector("#card-note").checked,
+			creationDate: document.querySelector("#card-creationDate").checked,
+			fields: document.querySelector("#card-fields").checked,
+			scrollableBio: document.querySelector("#card-scroll-bio").checked,
+		},
 		logging: document.querySelector("#logging").checked,
-		statusVisibility: document.querySelector("#status").checked,
-		notificationVisibility: document.querySelector("#notification").checked,
-		accountVisibility: document.querySelector("#account").checked,
-		conversationVisibility: document.querySelector("#conversation").checked,
 	});
 }
 
 function restoreOptions() {
 	async function setCurrentChoice(result) {
-		if (!result.statusVisibility) {
+		if (!result.proplate || !result.hoverCard) {
 			await defaultOptions();
 		} else {
+			document.querySelector("#proplates-enabled").checked = result.proplate.enabled || false;
 			document.querySelector("#logging").checked = result.logging || false;
-			document.querySelector("#status").checked = result.statusVisibility || false;
-			document.querySelector("#notification").checked = result.notificationVisibility || false;
-			document.querySelector("#account").checked = result.accountVisibility || false;
-			document.querySelector("#conversation").checked = result.conversationVisibility || false;
+			document.querySelector("#status").checked = result.proplate.statusVisibility || false;
```

Comment:
for consistency with the new cards-inputs prefix all id/name of the html elements for proplate settings?

Reply:
good point - will prefix them with `plate-`